CHANGELOG.md

```diff
@@ -6,6 +6,28 @@ All notable changes to [diagram-js](https://github.com/bpmn-io/diagram-js) are d
 
 _**Note:** Yet to be released changes appear here._
 
+* `FEAT`: do not scale context pad and popup menu by default
+
+### Breaking Changes
```

Comment:
Do we consider this a breaking change that warrants a major? I don't see it as "breaking" of either integrations or user flows.

Hence I'd rather make this a normal feature, link it accordingly, and ensure this PR is descriptive, i.e. contains the way to restore the old behavior.

Reply:
Depends on how you define "breaking". I agree that it will not break integrations, so we can ship it as a minor

Reply:
I updated the changelog and PR description accordingly

Reply:
Thanks!